Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -38,6 +38,11 @@ GITPATH="$(find $PACKAGES/gitformac.gvfs.installer/$GITVERSION -type f -name *.d
 # Now that we have a path containing the version number, generate GVFSConstants.GitVersion.cs
 $SCRIPTDIR/GenerateGitVersionConstants.sh "$GITPATH" $BUILDDIR || exit 1
 
+# If we're building the Profiling(Release) configuration, remove Profiling() for building .NET code
+if [ "$CONFIGURATION" == "Profiling(Release)" ]; then
+  CONFIGURATION=Release
```

Comment:
Shouldn't this be `DOTNETCONFIGURATION`?

Reply:
Oh I see, `DOTNETCONFIGURATION` hasn't been set yet. I would probably clean this up so we're deciding how to set `DOTNETCONFIGURATION` rather than changing `CONFIGURATION`

Reply:
We have to set DOTNETCONFIGURATION for the Debug/default case too so I'd rather have two separate conversions. One to reset 'Profiling(Release)' to Release and then another to unconditionally add '.Mac' to the configuration, regardless of if it's Debug/Release/malformed input.